udata/api/fields.py

```diff
@@ -4,7 +4,28 @@
 import pytz
 from dateutil.parser import parse
 from flask import request, url_for
-from flask_restx.fields import *  # noqa
+
+# Explicitly import all of flask_restx fields so they're available throughout the codebase as api.fields
+from flask_restx.fields import Arbitrary as Arbitrary
```

Comment:
Unsure about the `as` with same value?
```suggestion
from flask_restx.fields import Arbitrary as Arbitrary
```

Reply:
This is a "trick": it allows the linter to understand that this import is unused locally, but still needed.
As explained by the comment, we had a `from flask_restx.fields import *  # noqa` so the `flask_restx` fields were available as `api.fields`, but that would trip the linter:
- not being able to check if the imports were used, AND
- not being able to understand that the `flask_restx` fields used in the file were coming from the `import *`

Reply:
Fine with me for explicit import instead of `*` but I would use a `# noqa` that seems more explicit than this trick.